packages/sitecore-jss-react/src/components/FEaaSComponent.tsx

```diff
@@ -79,14 +79,18 @@ export const FEaaSComponent = (props: FEaaSComponentProps): JSX.Element => {
     return null;
   }
 
-  let data: string = null;
+  let data: { [key: string]: unknown } = null;
   if (props.params?.ComponentDataOverride) {
     // Use override data if provided
-    data = props.params.ComponentDataOverride;
+    try {
+      data = JSON.parse(props.params.ComponentDataOverride);
+    } catch (e) {
+      data = null;
+    }
   } else if (props.fields) {
     // Otherwise use datasource data (provided in fields)
     // FEaaS expects wrapping "_" as catch-all datasource template id
```

Comment:
Looks like FEAAS no longer requires the wrapping "_"? Can delete this obsolete comment then